### ISSUE TITLE
[Feature] detect helm chart errors during terraform plan

### ISSUE THREAD

**Issue by lots0logs** [rancher/terraform-provider-rancher2#1211]
### Is your feature request related to a problem? Please describe.
It's frustrating when errors in the helm chart belonging to `rancher2_app_v2` resources are not detected during terraform plan and only show up when helm upgrade fails during terraform apply. 

### Describe the solution you'd like
I think the provider should run helm upgrade with the `--dry-run` argument during the plan run so it can detect any errors and make the plan fail accordingly.

### Describe alternatives you've considered
I've considered never making errors in my helm charts but obviously that's not realistic :wink:

**Comment by matttrach** [comment]
Hello, thanks for the suggestion! sorry for the late response, is this still relevant?

**Comment by lots0logs** [comment]
Yeah, it's still relevant though its rare nowadays thanks to AI 🤷‍♂️
